src/System.Private.CoreLib/src/System/Reflection/Emit/DynamicILGenerator.cs

```diff
@@ -673,6 +673,7 @@ internal enum SecurityControlFlags
             CanSkipCSEvaluation = 0x8,
         }
 
+#pragma warning disable CS8609 // TODO-NULLABLE: Covariant return types (https://github.com/dotnet/roslyn/issues/23268)
```

Comment:
The base class annotation should be fixed instead.

Reply:
Fixed.